src/devices/DCMotor/DCMotor.csproj

```
@@ -11,6 +11,7 @@
     <Compile Include="DCMotor.cs" />
     <Compile Include="DCMotor3Pin.cs" />
     <Compile Include="DCMotor2PinNoEnable.cs" />
+	<Compile Include="DCMotor2PinWithBiDirectionalPin.cs" />
```

```suggestion
    <Compile Include="DCMotor2PinWithBiDirectionalPin.cs" />
```

(tab instead of spaces)